frontend/src/components/FacilityBooking/ViewSection.tsx

```diff
@@ -1,37 +1,47 @@
-import React, { useEffect } from 'react'
-import { useDispatch, useSelector } from 'react-redux'
+import React, { useEffect, useState } from 'react'
+import { useDispatch, useSelector, useStore } from 'react-redux'
 
 import { DailyContainer, MainContainer } from './BlockStyles'
 import ViewBlock from './ViewBlock'
 import { RootState } from '../../store/types'
 import HourBlocks from './HourBlocks'
 import CurrentTimeLine, { isToday } from './CurrentTimeLine'
 import { getTimeBlocks } from '../../store/facilityBooking/action'
+import { ViewBookingCard } from './ViewBookingCard'
+import { myBookingsStub } from '../../store/stubs'
+import { Booking } from '../../store/facilityBooking/types'
 
 const ViewScheduleBlock = () => {
   const { timeBlocks } = useSelector((state: RootState) => state.facilityBooking)
   const dispatch = useDispatch()
+  const [isViewBookingModalOpen, setIsViewBookingModalOpen] = useState(false)
+  const [viewBooking, setViewBooking] = useState<Booking>(myBookingsStub[0])
```

Comment:
would null or undefined be a better initial state? not sure if a stub is good in this case

Reply:
979468631